gateway/handle.go

```diff
@@ -261,8 +261,9 @@ func (gw *Handle) userWebRequestWorkerProcess(userWebRequestWorker *userWebReque
 // - user in the payload is not allowed to send events (suppressed)
 func (gw *Handle) getJobDataFromRequest(req *webRequestT) (jobData *jobFromReq, err error) {
 	var (
-		arctx    = req.authContext
-		sourceID = arctx.SourceID
+		arctx         = req.authContext
+		sourceID      = arctx.SourceID
+		destinationId = arctx.DestinationId
```

Comment:
it should be destinationID to follow Go's idiomatic capitalization
```suggestion
		destinationID = arctx.DestinationID
```